app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_for (current_user.active_relationships.build, remote: true) do |f| %>
```

Comment:
What is that space? This will cause problem.
How do you properly write a FORM_FOR?